backend/src/v5/models/projects.js

```diff
@@ -20,8 +20,14 @@ const { PROJECT_ADMIN } = require('../utils/permissions/permissions.constants');
 const db = require('../handler/db');
 const { templates } = require('../utils/responseCodes');
 
-const findProject = (ts, query, projection, sort) => db.find(ts, 'projects', query, projection, sort);
+const findProjects = (ts, query, projection, sort) => db.find(ts, 'projects', query, projection, sort);
 const findOneProject = (ts, query, projection) => db.findOne(ts, 'projects', query, projection);
+const updateOneProject = (ts, query, data) => db.updateOne(ts, 'projects', query, data);
+const updateProjects = (ts, query, data) => db.updateMany(ts, 'projects', query, data);
+
+Projects.addProjectModel = (ts, project, model) => updateOneProject(ts, { _id: project }, { $push: { models: model } });
```

Comment:
might be better naming to call these 2 `addModelToProject` and `removeModelFromProject`